### DISCUSSION TITLE
How to render as  javascript variable in qute template

### DISCUSSION THREAD

**Discussion by train860** [quarkusio/quarkus#27712] [Q&A] [answered]
Try to make a javascript variable from qute data like this:

var product = {data.object2Json};

but, always show "" as &quot there is any other ways but replace the  &quot by javascript?

[a comment by an automated account is not shown]

**Comment by mkouba** [comment, marked as answer]
Just use `{data.object2Json.raw}` or `{data.object2Json.safe}`. See also https://quarkus.io/guides/qute-reference#character-escapes.

**Reply by train860** [reply]
That's it, thanks
